classes/PlayerHandler.js

```diff
@@ -92,12 +92,12 @@ module.exports = class PlayerHandler {
 	 * Sends a localized message to the bound text channel.
 	 * @param {string} code The code of the locale string to be used.
 	 * @param {{title?: string, footer?: string, thumbnail?: string, additionalEmbeds?: EmbedBuilder[], files?: import('discord.js').FileOptions[], components?: import('discord.js').ActionRowBuilder[]}} [embedExtras] Extra data to be passed to the embed.
-	 * @param {boolean} [error] Whether or not the message is an error.
+	 * @param {"success"|"neutral"|"warning"|"error"} [type=neutral] Type of the message. Defaults to success.
 	 * @param  {...string} [args] Additional arguments to be passed to the locale string.
 	 * @returns {Promise<import('discord.js').Message>|Promise<import('discord-api-types/v10').APIMessage>|boolean} The message that was sent.
 	 */
-	async locale(code, embedExtras, error, ...args) {
+	async locale(code, embedExtras, type, ...args) {
```

Comment:
"Defaults to success", this method's `type` parameter has no default value.

Remove doc `@param` `[type=neutral]`, this method's parameter `type` has no default value.